experimental/torch_xla2/torch_xla2/export.py

```diff
@@ -3,10 +3,13 @@
 import copy
 from typing import Any, Dict, Tuple
 import torch
-from torch_xla2.ops import ops_registry
-from torch_xla2 import tensor
 from torch.utils import _pytree as pytree
-
+from torch_xla2 import tensor
+from torch_xla2.ops import ops_registry
+from jax.experimental import export
```

Comment:
IMO it would be preferable to import the module and refer to `export` by its fully qualified name. Same comment in other places, there are many `export` functions and `export` is a very generic name so having the full name helps with readability.

Reply:
done